diff --git a/patterns.js b/patterns.js
index ca8acec49c16b7d4eb7c74aaf253bea8d746d11e..ecacd15dcba232ad5980a29fd93092551ca3dacb 100644
--- a/patterns.js
+++ b/patterns.js
@@ -28,7 +28,8 @@ module.exports = options => {
         {
           shift: 0,
           block: true,
-          info: utils.hasDelimiters('end', options)
+          info: utils.hasDelimiters('end', options),
+          markup: (str) => !/^[`~]{3,}$/.test(str)
         }
       ],
       transform: (tokens, i) => {
